### ISSUE TITLE
Incorrect validation occuring for some parameters

### ISSUE THREAD

**Issue by briangleeson** [IBM/openapi-validator#489]
Version: `ibm-openapi-validator@0.91.0`

The validator comes back with this warning that seems to be invalid:
```
 Message :   $refs to schemas should start with '#/components/schemas/'                                                                                   
 Path    :   paths./tekton_pipelines/{pipeline_id}/properties.parameters.0.$ref                                                                           
 Line    :   1022
```
This is coming up for 3 parameters in our API. The full yaml file can be seen [here](https://github.ibm.com/org-ids/pipeline-ui/blob/16c124641c37e8778bf0adf1fde763503ad5a01a/tekton/controllers/spec/v2/openapi.yaml)

For example this one gets the warning:

```
/tekton_pipelines/{pipeline_id}/properties:
  parameters:
    - $ref: '#/components/parameters/pipelineId'

```
```
components:
  parameters:
    pipelineId:
      description: The Tekton pipeline ID
      in: path
      name: pipeline_id
      required: true
      schema:
        $ref: '#/components/schemas/UUID'
      example: 94619026-912b-4d92-8f51-6c74f0692d90
```
We're using this parameter for all of our APIs in the same way, but it only gives a warning for 2 cases of `pipelineId` parameter and one case of `triggerId` parameter.

This appears to be a bug in the validator related to validating parameter references
